eds4jinja2/adapters/local_sparql_ds.py

```diff
@@ -29,18 +29,23 @@ def __init__(self, filename):
     def __reduce_bound_triple_to_string_format(self, dict_of_bound_variables: dict):
         return {str(k): str(v) for k, v in dict_of_bound_variables.items()}
 
-    def with_query(self, sparql_query: str) -> 'RDFFileDataSource':
+    def with_query(self, sparql_query: str, **kwargs) -> 'RDFFileDataSource':
```

Comment:
We can unpack, indeed the kwargs and have an arbitrary number of parameters, the problem is that we will not really know which ones to check in the function. Also we need to always remember to unpack the dicts as parameters. 

We better pass that dict as a parameter.